app/components/Onboarding/Onboarding.js

```diff
@@ -161,7 +161,10 @@ const Onboarding = ({
         return (
           <FormContainer
             title="Welcome back!"
-            description="Enter your wallet password or create a new wallet"
+            description={`It looks like you have already a wallet
+              ${Boolean(initWalletProps.loginProps.existingWalletDir) &&
+                `(we found one at \`${initWalletProps.loginProps.existingWalletDir}\`)`}.
+            Please enter your wallet password to unlock it.`}
```

Comment:
nit: Could look better with a newline before the last sentence.

Reply:
Looked deeper and not that easy to breakup given the current setup of FormContainer, so maybe not worth it.